kubeadm/scripts/PrepareNode.ps1

```diff
@@ -46,6 +47,7 @@ $kubeletBinPath = "$global:KubernetesPath\kubelet.exe"
 mkdir -force "$global:KubernetesPath"
 $env:Path += ";$global:KubernetesPath"
 [Environment]::SetEnvironmentVariable("Path", $env:Path, [System.EnvironmentVariableTarget]::Machine)
+[Environment]::SetEnvironmentVariable("NODE_IP", $NodeIP, [System.EnvironmentVariableTarget]::Machine)
```

Comment:
instead of taking an flag to the ps1 script, which then sets an env. variable which is then passed to the kubelet, why not pass the `--node-ip=....` to the kubelet only if the user has set `NODE_IP` outside of the ps1 - e.g. in their environment?

Reply:
or just pass the value of the ps1 flag `NodeIP` directly to `--node-ip` to the kubelet.
if `--node-ip` equals `""` the kubelet should apply it's detection.

Reply:
Hi @neolit123 ,
- Firstly, consider the kubelet runs as service and user is easy to update `NODE_IP` by setting environment. So I think get the param from environment when booting kubelet is good.
- Basic on the above choice, ps1 just need to pass the flag NodeIP to environment. If we pass the flag to kubelet directly, user will must change the  `StartKubelet.ps1` when they want to change the node ip.

How do you think?

Reply:
ok, my vote is to go with the first option in this PR.

but also please add a note under `.DESCRIPTION`:
> You can use set the "NODE_IP" environment variable to configure the node IP for the kubelet

Reply:
Thanks Lubomir. Update the `.DESCRIPTION` per your comment.

Reply:
but i think this is not needed anymore, right?
```
[Environment]::SetEnvironmentVariable("NODE_IP", $NodeIP, [System.EnvironmentVariableTarget]::Machine)
```
because the user is supposed to set the env variable now.

Reply:
Yes, correct. Remove the flag from ps1.